test/integration/patients/list/patients-all.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'underscore';
+
+context('patient sidebar', function() {
+  specify('display patient data', function() {
+    cy
+      .server()
+      .routePatient()
+      .routePatients(fx => {
+        fx.data[0].id = 1;
+        fx.data[0].attributes = {
+          first_name: 'First',
+          last_name: 'Last',
+        };
+
+        _.each(fx.included, inc => {
+          if (inc.type === 'groups') {
+            inc.attributes = {
+              name: 'Group Name',
+            };
+          }
+        });
+
+        return fx;
+      })
+      .visit('/patients/all')
+      .wait('@routePatients');
+
+    cy
+      .get('.app-frame__content')
+      .should('contain', 'First Last')
```

Comment:
These `should`s should be able to move between 36 & 37.. that might require pushing a new group to includes and the making the relationship on `fx.data[0]`.  It'd be similar to https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/integration/patients/patient-sidebar.js#L98   Then we can just modify the group for the test row and leave the rest random